gen/aws/templates/cloudformation.json

```diff
@@ -71,10 +71,10 @@
         "stable": "ami-05bc0164"
       },
       "us-west-1": {
-        "stable": "ami-27553a47"
```

Comment:
We need to at the very least give notice to support that we're changing the base requirements if we're going to update our Docker requirement. We also need to get from support if upgrading docker will be a dealbreaker for any customers (We really don't want to strand customers on old DC/OS).

As far as upgrading CoreOS, we need to do it in all regions/AMIs if we're doing it. And do it for other providers than AWS. Also note we intentionally haven't bumped CoreOS in a while: https://dcosjira.atlassian.net/browse/DCOS-38. We need to resolve that issue before we can update / change the CoreOS version.